### PR TITLE
Fix Indentation of Numbered Bullet Points

### LINKED ISSUE #140: Sub heading numbers are not properly aligned in Graphical View page
**Description:**
$Subject in the "Launching the Graphical View" section of the Graphical View page [1]

[1] https://ballerina.io/learn/tools-ides/vscode-plugin/graphical-editor

image

**Suggested Labels:**

**Suggested Assignees:**

**Affected Product Version:**

**OS, Browser, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
This is to fix the indentation issue in the numbered bullet points in the "Graphical Editor" page of the VSCode extension docs.

> Fixes #140

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: learn/tools-ides/vscode-plugin/graphical-editor.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ redirect_from:
 
 # Graphical View
 
-A rich set of Visualization tools will immensely enhance your development experience especially in the integration space. 
+A rich set of visualization tools will immensely enhance your development experience especially in the integration space.
```
Comment (anupama-pathirage) [inline learn/tools-ides/vscode-plugin/graphical-editor.md]: Shall we fix the same in the other 3 files as well?

Comment (praneesha) [reply learn/tools-ides/vscode-plugin/graphical-editor.md]: Fixed.

Comment (anupama-pathirage) [reply learn/tools-ides/vscode-plugin/graphical-editor.md]: Still cannot see this fix in the other files. Please recheck.

Comment (praneesha) [reply learn/tools-ides/vscode-plugin/graphical-editor.md]: Fixed.

**Pull request merged**